core/src/execution/datafusion/expressions/cast.rs

```diff
@@ -19,6 +19,7 @@ use std::{
     any::Any,
     fmt::{Debug, Display, Formatter},
     hash::{Hash, Hasher},
+    num::Wrapping,
```

Comment:
I wasn't aware of `Wrapping`. TIL.

Reply:
Wonder if this is also faster since there is no overflow check?

Reply:
I think we usually use some methods like `wrapping_add`, etc.